### PR TITLE
Feature: Update to newer icon codes (pt.1/2)

### PR DESCRIPTION
**Pull request by Jay-o-Way**
Visually there is no change

**Resolved / Related Issues**
- [ ] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Related #10783

**Validation**
How did you test these changes?
- [ ] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?
   1. Open app ...
   2. Click settings button ...

**Screenshots (optional)**
Add screenshots here.

### TO-DO

The following either
- can't find exact replacement, or
- double check context

image

### CODE REVIEW

**Comment by Jay-o-Way** [comment]
@yaira2 I'll look again soon. Input on the remaining icons (see image ⬆️) is welcome. If you have access to the branch, you're welcome to add on this.

**Comment by Jay-o-Way** [comment]
@yair100 I don't know what's going on here, but there's something seriously going wrong...
Error list

File: src/Files.App/Views/Settings/AppearancePage.xaml [hunk 1]
```diff
@@ -161,7 +161,7 @@
 			<!--  Compact Spacing  -->
 			<local:SettingsBlockControl Title="{helpers:ResourceString Name=UseCompactSpacing}" HorizontalAlignment="Stretch">
 				<local:SettingsBlockControl.Icon>
-					<FontIcon Glyph="&#xE14C;" />
+					<FontIcon Glyph="&#xE8FD;" />
```
[the remark this answers is not in the document]
Comment (Jay-o-Way) [reply src/Files.App/Views/Settings/AppearancePage.xaml]: @yair100 You sure about that? 'cause this is what is see:
icons

[the remark this answers is not in the document]
Comment (Jay-o-Way) [reply src/Files.App/Views/Settings/AppearancePage.xaml:164]: Note: The **Fluent** icon (as on Windows 11, https://learn.microsoft.com/windows/apps/design/style/segoe-fluent-icons-font) simply _looks_ different than the **MDL2** font.

[the remark this answers is not in the document]
Comment (yaira2) [reply src/Files.App/Views/Settings/AppearancePage.xaml:160]: You can use Character Map UWP to compare the current icon and the one you're placing it with.

[the remark this answers is not in the document]
Comment (Jay-o-Way) [reply src/Files.App/Views/Settings/AppearancePage.xaml:160]: 1. You could and should have provided an image to back-up your claim.
2. No, I should _not_ use **third-party** apps. You can look at the images I provided above, or you can use **WinUI 3 Gallery** to see for yourself.
3. For the sake of argument, I did check with Character Map, and guess what? They are **identical**.
   image

[the remark this answers is not in the document]
Comment (yaira2) [reply src/Files.App/Views/Settings/AppearancePage.xaml:160]: It's probably a difference between Windows 10 and Windows 11 (as you originally suggested). I'll try to find a good icon that properly reflects the setting.

I want to also remind you of the code of conduct, I'm sure you didn't mean it this way, but your tone is coming across as a little rude. This is a learning process for everyone (myself included) and we're all in this together 🙂

[the remark this answers is not in the document]
Comment (Jay-o-Way) [reply src/Files.App/Views/Settings/AppearancePage.xaml:160]: Meh, you were also right. I can be a bit too direct sometimes. Sorry.

[the remark this answers is not in the document]
Comment (yaira2) [reply src/Files.App/Views/Settings/AppearancePage.xaml:160]: In Windows 11, the current icon looks like this
image

File: src/Files.App/Views/Settings/AppearancePage.xaml [hunk 2]
```diff
@@ -157,7 +157,7 @@
...
```
Comment (yaira2) [inline src/Files.App/Views/Settings/AppearancePage.xaml:160]:
```suggestion
					<FontIcon Glyph="&#xE62B;" />
```

**Review by yair100 — DISMISSED** [review]
LGTM

**Comment by yaira2** [comment]
@Jay-o-Way can you resolve the merge conflicts?

**Comment by Jay-o-Way** [comment]
We're not ready yet. I left a few undone, because I would like for you to double check what they should be replaced with.
image

**Comment by yaira2** [comment]
> We're not ready yet. I left a few undone, because I would like for you to double check what they should be replaced with.

We can cover those in another PR, can you send them in text form so I can copy/paste?

**Comment by Jay-o-Way** [comment]
> We can cover those in another PR, can you send them in text form so I can copy/paste?

I could, but there's another way:
Search in files, query = `xE[1-5]` with regular expression _on_

**Pull request merged**